src/plugins/diagram/structure_plugin.js

```diff
@@ -303,20 +303,20 @@ define( ['structure_custom',
 
       // plot diagram initially with empty values
       diagram.empty();
-      data.plot = $.plot(diagram, [], options);
+      var plot = $.plot(diagram, [], options);
```

Comment:
Yes, data is shared between the popup and the non-popup version of the diagram. plot is diagram specific and can not be shared, so i needed to factor it out of the data structure.